js-java-async-helidon/pom.xml

```diff
@@ -70,8 +96,29 @@
 
         <dependency>
             <groupId>org.graalvm.sdk</groupId>
-            <artifactId>graal-sdk</artifactId>
+            <artifactId>collections</artifactId>
+            <version>${graalvm.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.graalvm.sdk</groupId>
+            <artifactId>nativeimage</artifactId>
+            <version>${graalvm.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.graalvm.sdk</groupId>
+            <artifactId>word</artifactId>
+            <version>${graalvm.version}</version>
+        </dependency>
+        <dependency> 
+            <groupId>org.graalvm.polyglot</groupId> 
+            <artifactId>polyglot</artifactId> 
+            <version>${graalvm.version}</version>
+        </dependency>
+        <dependency> 
+            <groupId>org.graalvm.polyglot</groupId> 
```

Comment:
@chumer , we upgraded this demo to latest GraalVM. At run time, this app fails with: 
```
Caused by: java.lang.IllegalArgumentException: A language with id 'js' is not installed. Installed languages are: [].
```
Please see the GitHub action log: https://github.com/graalvm/graalvm-demos/actions/runs/7166005885/job/19509126528 
What are we missing?